docs/docs/guides/How to Add Digest to In-App Notifications.md

```diff
@@ -0,0 +1,293 @@
+
+### What is a Digest Notification?
+
+Often times when you associate notifications with user activity, the end user can be bombarded with messages because of the nature of the activity. Take for example the case of commenting in the context of a social media app. If you were to send a notification to a user for every comment they receive, and they happen to receive 100 comments, it could lead to user fatigue since you would have to send 100 messages.
+
+This is where digest notifications come into the picture!
+
+A digest notification is a notification that consolidates information from several notifications into one and delivers that notification to the end user instead of several separate messages.
+
+### How does Digest Notification Work?
+
+Novu has a built-in digest engine that collects multiple trigger events, aggregates them into a single message and delivers it to the subscriber. You can use the digest engine by adding a ‘digest node’ to your workflow in the workflow editor in the [Novu dashboard](https://web.novu.co/workflows). If you want to learn more about it, [this](https://docs.novu.co/platform/digest/) is a great place to start.
+
+Let’s see it in action now!
+
+## Add Digest Notification To An App
+
+To get started with this, you need the following:
+
+- A Novu account. [Sign up for free](http://web.novu.co) if you don’t have one yet.
+- A working React development environment.
+
+## Setting things up in Novu
+
+Once, you have these, you need to follow the following steps:
+
+1. Head over to the Novu Dashboard.
+2. Click “Workflows” on the left sidebar of your Novu dashboard.
+3. Click the “Create Workflow” button on the top right:
+
+![Create workflow on Novu web dashboard](https://res.cloudinary.com/dxc6bnman/image/upload/v1688127676/guides/SCR-20230630-pqdm_z5npqe.png)
+
+Once you click the ‘create workflow’ button, you’ll see a dropdown. Select ‘blank workflow’ from the dropdown:
+
+![select blank workflow in dropdown](https://res.cloudinary.com/dxc6bnman/image/upload/v1688131596/guides/Screenshot_2023-06-25_at_10.54.37_PM_cstufw.png)
+
+You’ll now be taken to the workflow editor:
+
+![workflow editor in Novu workflow editor](https://res.cloudinary.com/dxc6bnman/image/upload/v1688131696/guides/Screenshot_2023-06-25_at_10.56.11_PM_b7fc7i.png)
+
+Once here, you can add the channels you want to use for sending notifications and configure them. For this guide, we’ll use the ‘In-App’ channel. You’ll also see the ‘Digest’ action on the right sidebar.
+
+Note: Note: Each node that is added below the digest node will only be triggered after the specified time interval
+
+For example, in our case, say we want the In-App notification to be sent after every 6 hours. Then we’ll add the ‘digest’ action below the ‘trigger node’ and the ‘in-app’ channel node below the ‘digest’ node as shown below:
```

Comment:
Please rewrite: `Then we’ll add the ‘digest’ action below the ‘trigger node’ and the ‘in-app’ channel node below the ‘digest’ node as shown below:`

to


=> Next, add the 'digest' action below the 'trigger node', and add the 'in-app' channel node below the 'digest' node as shown below: